es/opendistro/api.py

```diff
@@ -88,12 +91,15 @@ def __init__(
         if user and password:
             self.es = Elasticsearch(self.url, http_auth=(user, password), **self.kwargs)
         else:
-            self.es = Elasticsearch(self.url, **self.kwargs)
-
-    def _aws_auth(self, aws_access_key, aws_secret_key, region):
-        from requests_4auth import AWS4Auth
-
-        return AWS4Auth(aws_access_key, aws_secret_key, region, "es")
+            awsauth = self._aws_auth()
+            self.es = Elasticsearch(
+                [{'host': host, 'port': port}], http_auth=awsauth, use_ssl=True, verify_certs=True,
```

Comment:
`use_ssl=True` and `verify_certs=True` makes sense security wise but on same cases it's nice the let them be configured differently (self signed certs, dev envs, etc)